configure.py

```diff
@@ -319,6 +319,15 @@ def get_var(environ_cp,
     question += ' [y/N]: '
 
   var = environ_cp.get(var_name)
+  if var is not None:
+    _var_content = var.strip().lower()
+    if _var_content in ['1', 'y', 'true']:
+      var = True
+    elif _var_content in ['0', 'n', 'false']:
+      var = False
+    else:
+      var = None
```

Comment:
In this case, should you error out instead?

Reply:
Made changes to throw a `UserInputError` in this case. Thanks!